init_test_data.sql

```diff
@@ -2,31 +2,38 @@ USE data;
 
 -- Create initial tables
 
+CREATE TABLE geography (
+    geography_id serial primary key,
```

Comment:
thought on this. for readability, should we have the abbreviation be the PK so we don't need to join against this table for any visual scan of core_data? and is there any reason not to?

Reply:
I went back and forth on that repeatedly since last night. Making the abbreviation the PK makes things so much more human readable, but starts to fail if we want to start tracking other non-state geographies in the future. On balance though, I think I'm persuaded that readability is more important than something we may never need (being able to scroll through the data table without memorizing state ids sounds important for our sanity).